roles/iam-kms/README.md

```diff
@@ -0,0 +1,25 @@
+#iam-kms
+
+Simple role to create/update master keys and relevant aliases, the role will 
+only get executed if the `kms_data` key is present in the meta-data
+
+####Example meta-data and output
+```yaml
+opg_data:
+  project: foo
+  environment: bar
+  
+kms_data:
+  - key_description: baz
```

Comment:
Isn't this a suffix?

Reply:
Possibly, but I use it in the tags and in the alias as well... bit of a hard one to define

Reply:
Maybe just name the variable better to let us know it's a suffix, or just part of the naming scheme